provider/aws/dist/rack.json

```diff
@@ -1698,11 +1698,11 @@
           }
         ],
         "Role": { "Fn::GetAtt": [ "ServiceRole", "Arn" ] },
-        "TaskDefinition": { "Ref": "RackWebTasks" }
+        "TaskDefinition": { "Ref": "WebTasks" }
       }
     },
-    "RackMonitor": {
```

Comment:
I don't think RackMonitor needs this since it doesn't have ports, so doesn't have the same capacity problems as RackWeb. Were you just changing it to keep the naming scheme consistent?

Wondering if we could keep this as RackMonitor and call the other one RackAPI.